### DISCUSSION TITLE
Import credit card transactions from CSV

### DISCUSSION THREAD

**Discussion by davxiao** [firefly-iii/firefly-iii#8644] [Firefly III]
I wanted to use firefly to analyze credit card spending only, and so I tried creating "credit card" role asset account but when importing it is still shown "deposit" and "withdrawal" as opposed to "purchase" and "credit"
I also tried to create credit card in expense account, but from the importer the account drop down only allows me to select account from asset account type.
Any pointer?

**Comment by JC5** [comment]
Hi!

I think I can answer this: 

- The terminology will not change when you set the role to credit card.
- Credit cards are also available in the dropdown regardless of the role.
